Dockerfile

```diff
@@ -26,8 +29,7 @@ ENV PETSC_SETUP_ARGS --with-cc=gcc \
 	--download-exodusii \
 	--download-fftw \
 	--download-hdf5 \
-	--download-metis \
-	--download-ml \
+	--download-metis \	
```

Comment:
I think that in the next release we will need  --download-opencascade  if we have egads.

Reply:
I'll wait for the next release and merge then.